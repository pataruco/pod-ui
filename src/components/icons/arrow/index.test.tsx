import { shallow } from 'enzyme';
import React from 'react';
import Arrow from './index';

describe('Arrow', () => {
  it('renders', () => {
    expect(shallow(<Arrow />)).toMatchSnapshot();
  });
});
