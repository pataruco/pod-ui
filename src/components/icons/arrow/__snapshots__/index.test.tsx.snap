// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow renders 1`] = `
<svg
  className="svg"
  viewBox="0 0 100 100"
>
  <polygon
    points="50,16 85,85 15,85 50,16"
  />
</svg>
`;
